uber/model_checks.py

```diff
@@ -37,6 +37,30 @@ def has_email_address(account):
 
 
 @validation.Group
```

Comment:
Daaangit.

On a related note, I really want a `@prereg_validation` class of some sort, so we don't even have to make this trade-off. I may look into seeing how easy that would be to do.